Assets/Scripts/Framework/ScoreManager.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using Framework;
+using TMPro;
+using UnityEngine;
+
+namespace FrameWork
+{
+    public class ScoreManager : Singleton<ScoreManager>
+    {
+        public int TotalScore { get; private set; }
+        
+        [SerializeField] private TextMeshProUGUI scoreText;
```

Comment:
```suggestion
        [SerializeField] private TextMeshProUGUI scoreText;

        public int TotalScore { get; private set; }
```
![image](https://github.com/Team-Swamp/IceBites/assets/70578065/72221f6f-3714-4ee3-a345-2a912a047f3f)